### ISSUE TITLE
Error in writer.py

### ISSUE THREAD

**Issue by ReshmaAnnaThomas** [thepetabyteproject/your#36]
**Describe the bug**
your_writer.py gives a value error.

**To Reproduce**

```
your_writer.py -f *fits -t fil -o ./cands -r -sksig 4 -sgsig 4 -sgfw 15 -zero_dm_subt 

Traceback (most recent call last):
  File "/home/rat0022/anaconda3/envs/new/bin/your_writer.py", line 4, in <module>
    __import__('pkg_resources').run_script('your==0.5.1', 'your_writer.py')
  File "/home/rat0022/anaconda3/envs/new/lib/python3.6/site-packages/pkg_resources/__init__.py", line 667, in run_script
    self.require(requires)[0].run_script(script_name, ns)
  File "/home/rat0022/anaconda3/envs/new/lib/python3.6/site-packages/pkg_resources/__init__.py", line 1464, in run_script
    exec(code, namespace, namespace)
  File "/home/rat0022/anaconda3/envs/new/lib/python3.6/site-packages/your-0.5.1-py3.6.egg/EGG-INFO/scripts/your_writer.py", line 70, in <module>
    w.to_fil()
  File "/home/rat0022/anaconda3/envs/new/lib/python3.6/site-packages/your-0.5.1-py3.6.egg/your/writer.py", line 167, in to_fil
    if nsamps % interval != 0:
ValueError: The truth value of an array with more than one element is ambiguous. Use a.any() or a.all()
```

**Comment by KshitijAggarwal** [comment]
Debugging this, there is a logic error in calculation of `nloops` and `nsamps` as well.

**Comment by KshitijAggarwal** [comment]
Fixed with 4ed4dae5925e200fe851563af403f041c32de424
